### PR TITLE
#105 - Academy Quote Block

### PR DESCRIPTION
**Pull request by jhancock532**
### [Link to Ticket On Codebase](https://projects.torchbox.com/projects/contentful-careers-page/tickets/105)

I've updated the Quote component so that the role is now optional: I renamed role to position as the linter thought it was an accessibility error (`role` shouldn't be defined on `quote`).

### How to Test

Go to the academies page on the staging site, linked below:

[Link to preview site page with component](https://careers-4jl1sarvl-careers-site.vercel.app/careers/torchbox-academy)

<details>
  <summary>Screenshots</summary>
image
image
</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
- [x] Tests and linting passes.
- [x] Consider updating documentation. If you don't, tell us why.
- [x] If relevant, list the environments / browsers in which you tested your changes.

### CODE REVIEW

File: styles/pages/TorchboxAcademy.module.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.container {
+    max-width: 1440px;
```
Comment (Morsey187) [inline styles/pages/TorchboxAcademy.module.scss:2]:
```suggestion
    max-width: $site-width;
```

File: styles/pages/TorchboxAcademy.module.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    margin: 0 auto;
+}
+
+.textContainer {
+    margin: 0 auto $grid * 3 auto;
+    max-width: 755px;
```
Comment (Morsey187) [inline styles/pages/TorchboxAcademy.module.scss:8]:
```suggestion
    max-width: $content--narrow;
```
Would be good to update the other textContainer classes with this as well.

**Review by Morsey187 — APPROVED** [review]
LGTM! Happy for you to merge after updating to use the SCSS variables mentioned below.

File: styles/pages/TorchboxAcademy.module.scss (hunk 1 above)
Comment (jhancock532) [reply styles/pages/TorchboxAcademy.module.scss:2]: Thanks for the spot!

**Pull request merged**